### ISSUE TITLE
Support LUT Mods

### ISSUE THREAD

**Issue by effs** [E1337Kat/cyberpunk2077_ext_redux#31]

**Comment by Bladehawke** [comment]
As far as I can tell, there are only three LUT mods on Nexus, while we could write a handler for putting them into the folder they go into, in order to *use* them, the user actually needs to know the filenames and configure Reshade to use them.  Is this something we really need to support with a blind installation?

**Comment by effs** [comment]
Yeah, this is a good point. Maybe we just want different messaging for the install anyway prompt?
